libobs/obs-properties.h

```diff
@@ -428,7 +428,7 @@ EXPORT const char *obs_property_button_url(obs_property_t *p);
 
 #ifndef SWIG
 OBS_DEPRECATED
-EXPORT enum obs_text_type obs_proprety_text_type(obs_property_t *p);
+EXPORT enum obs_text_type obs_property_text_type(obs_property_t *p);
```

Comment:
This is an API break, the typoed function is intentionally there for backward compatibility and marked deprecated.